### PR TITLE
[WIP] Initial cli tests

### PR DESCRIPTION
**Pull request by vvscode**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ var browserslist = require('./');
 var pkg          = require('./package.json');
 var args         = process.argv.slice(2);
 
+
```
Comment (ai) [inline cli.js]: Oops, you add unnecessary newline ;)

File: npm-debug.log [hunk 2]
```diff
@@ -0,0 +1,46 @@
+0 info it worked if it ends with ok
```
Comment (ai) [inline npm-debug.log]: You commented also unnecessary `npm-debug.log`. I added it to `.gitignore` a4342a3, but you should check pull request diff next time.

File: test/cli.test.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
+const pkg = require('../package.json');
+const CLI_PATH = '../cli';
+
+let write;
```
Comment (ai) [inline test/cli.test.js]: `let` and `const` doesn‘t work in Node.js 4

**Comment by ai** [comment]
I added first review, fix it first. Tomorrow I will look into tests (I am drunk right now).

But right now I think mocking is too complicated. There is a better way:

1. Call real CLI by `require('child_process').spawn`.
2. Compare it with `browserslist()` function result. In this case you don’t need to mock CLI data.

**Comment by vvscode** [comment]
But such tests will be slow - is it ok? also they will work with network ( if I understand how browserlist works ). is it normal for tests?

**Comment by ai** [comment]
> But such tests will be slow - is it ok? 

It is OK :).

> also they will work with network

Nope, Browserslist doesn’t use network. It [has](https://github.com/ai/browserslist/blob/master/package.json#L13) own copy of Can I Use database.

**Pull request closed without merging**